components/host-dashboard/AddFundsModal.js

```diff
@@ -109,6 +111,22 @@ const addPlatformTipMutation = gqlV2/* GraphQL */ `
   }
 `;
 
+export const budgetSectionQuery = gqlV2/* GraphQL */ `
+  query BudgetSection($slug: String!, $limit: Int!, $kind: [TransactionKind]) {
+    transactions(account: { slug: $slug }, limit: $limit, hasExpense: false, kinds: $kind) {
+      ...TransactionsQueryCollectionFragment
+    }
+    expenses(account: { slug: $slug }, limit: $limit) {
+      totalCount
+      nodes {
+        ...ExpensesListFieldsFragment
+      }
+    }
+  }
+  ${transactionsQueryCollectionFragment}
+  ${expensesListFieldsFragment}
+`;
+
```

Comment:
Duplicated, import this from `collective-page/graphql/queries.js` after you move the other one.

Reply:
Corrected this. Thanks. 👍🏽